src/Haskell/Ide/Engine/Plugin/HieExtras.hs

```diff
@@ -82,8 +82,8 @@ getDynFlags = ms_hspp_opts . pm_mod_summary . tm_parsed_module
 
 -- ---------------------------------------------------------------------
 
-data NameMapData = NMD
-  { inverseNameMap ::  !(Map.Map Name [SrcSpan])
+newtype NameMapData = NMD
+  { inverseNameMap :: Map.Map Name [SrcSpan]
```

Comment:
You have removed a strictness annotation here. This is probably not intended.

Reply:
See also https://github.com/ndmitchell/hlint/issues/548

Reply:
My bad. I changed it back.

Reply:
For the time being I added annotations to ignore this use of `data` and the compatibility lambdas.